crates/common_utils/src/events.rs

```diff
@@ -57,6 +57,8 @@ pub enum ApiEventsType {
     Events {
         merchant_id_or_profile_id: String,
     },
+    PaymentMethodCollectLink,
+    PayoutLink,
```

Comment:
Is there any id that you want to store along with these events for e.g collect_link id etc